src/common/reports/results_test.go

```diff
@@ -118,10 +118,11 @@ func TestResultsGeneration(t *testing.T) {
 	t.Run("Test CLI output structure", func(t *testing.T) {
 		ReportServiceInst.CreateReport()
 
-		output := utils.CaptureOutput(ReportServiceInst.PrintToStdout)
+		output := utils.CaptureOutput(ReportServiceInst.PrintToStdout(noColorBool bool))
```

Comment:
@goody-cvs Should this be like so:
```suggestion
		output := utils.CaptureOutput(ReportServiceInst.PrintToStdout(false))
```
?